src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age !== undefined && (
+      <p className="Person__age">I am {`${person.age}`}</p>
+    )}
+    {person.isMarried === true && person.sex === 'm' && (
+      <p className="Person__partner">{`${person.partnerName}`} is my wife</p>
+    )}
+    {person.isMarried === true && person.sex === 'f' && (
+      <p className="Person__partner">{`${person.partnerName}`} is my husband</p>
+    )}
+    {person.isMarried === false && (
+      <p className="Person__partner">I am not married</p>
+    )}
```

Comment:
Looks too complicated, create a separate variable for person partner and use it here. You can also have only one `<p> `tag and change only it's content

`const partner = person.sex === 'm' ? 'wife' : 'husband'`